File: src/api/Models/Booking.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RentAPI.Models
{
    public class Booking
    {
        [Key] public int Id { get; set; }
        [Column("car_id")]
        public int CarId { get; set; }
        [Column("customer_id")]
        public int CustomerId { get; set; }
        [Column("pick_up_date")]
        public DateTime PickUpDate { get; set; }
        [Column("return_date")]
        public DateTime ReturnDate { get; set; }
    }
}

File: src/model/booking/Booking.java
package model.booking;

import java.util.Date;

import model.DatabaseRecord;

public class Booking extends DatabaseRecord {
    private final Integer _carId;
    private final Integer _customerId;
    private final Date _pickUpDate;
    private final Date _returnDate;

    public Integer GetCarId() {
        return _carId;
    }

    public Integer GetCustomerId() {
        return _customerId;
    }

    public Date GetPickUpDate() {
        return _pickUpDate;
    }

    public Date GetReturnDate() {
        return _returnDate;
    }

    public Booking(Integer id, Integer carId, Integer customerId, Date pickUpDate, Date returnDate) {
        super(id);
        this._carId = carId;
        this._customerId = customerId;
        this._pickUpDate = pickUpDate;
        this._returnDate = returnDate;
    }

    @Override
    public String toString() {
        return "Booking : {\n\tid=" + GetId() + "," + "\n\tcar_id=" + _carId + "," + "\n\tcustomer_id=" + _customerId
                + "," + "\n\tpick_up=" + _pickUpDate.toString() + "\n\treturn=" + _returnDate.toString() + "\n}";
    }

}
